types/storybook__addon-options/index.d.ts

```diff
@@ -1,10 +1,13 @@
-// Type definitions for @storybook/addon-options 3.2
+// Type definitions for @storybook/addon-options 4.0
```

Comment:
If you think the types for version 3 will continue to be updated, please follow the [directions for updating to a new major version](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version). No worries if not, just want to make sure before merging this.

Reply:
Good question ;) I just checked and no, there will never be a new 3.x release so we can move type def version to 4.0